upgrade_tests/test_existance_relations/test_architectures.py

```diff
@@ -5,14 +5,16 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: System
 
 :CaseComponent: CLI
 
-:TestType: NonFunctional
+:TestType: nonfunctional
 
 :CaseImportance: High
 
+:SubType1: installability
+
```

Comment:
After discussion with @elyezer and @omaciel , We decided to keep it installability.